### ISSUE TITLE
Some items not shown in multiselect

### ISSUE THREAD

**Issue by juxeii** [petereon/beaupy#57]
I have a really weird issue with `select_multiple`.
Consider this snippet:
```
colored_options = [
        '1',
        '2',
        '3',
        '4',
        '5',
        '6',
        '7',
        '8']

print(colored_options)
print(len(colored_options))
select_multiple(colored_options)
```
In container A, in a **python vscode terminal**, everything works as expected:
![image](https://user-images.githubusercontent.com/4497675/228280948-a6fd697d-923a-4786-99ca-2fb1bed11906.png)

In another container, in a **bash terminal**, I only ever see 5 items:
![image](https://user-images.githubusercontent.com/4497675/228274358-e1bcb947-4769-49c5-bd5c-c599ab9d7d16.png)

Why is the bash terminal only showing 5 items?
How can I debug this?

**Comment by juxeii** [comment]
Version:
`3.3.0`
Working container OS: 
```
NAME="Red Hat Enterprise Linux Server"
VERSION="7.9 (Maipo)"
```
Not working container OS:
```
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
```

**Comment by petereon** [comment]
Could you please verify it's not 3.5.0 on the Debian? In 3.5.0 I have been making changes that could have introduced this bug. 

Sorry, I am not at the computer atm so I can't verify.

**Comment by petereon** [comment]
Thanks for correction. Fix should be reasonably easy, I'll release **3.5.1** fixing this later today.

**Comment by juxeii** [comment]
It is working again, thank you for your fast support!

**Comment by petereon** [comment]
That's great to hear!

**Comment by juxeii** [comment]
I think there is a followup error on this:
I have a list with 11 entries, but with the up and down keys I can only navigate to the fifth element. Pressing down from there then goes back up to the first entry.
Since this happens to the fifth element, I guess this error is related to the other issue.

**Comment by petereon** [comment]
You are correct. I will be taking a look momentarily

**Comment by petereon** [comment]
Should be fixed, I am waiting for CI and then I'll merge and release. Note to self: I should really start doing beta releases :sweat_smile:

**Comment by petereon** [comment]
Okay, released as version `3.5.2`, please see the GitHub Releases or PyPI

**Comment by juxeii** [comment]
Thx, it is working.
